Definitions/Chord

```diff
@@ -0,0 +1,29 @@
+class Chord:
+    def __init__(self, symbol, degrees, next, inversion):
+        self.symbol = symbol
+        self.degrees = degrees
+        self.next = next
+        self.inversion = inversion
+        
+    def hasMelodicNote(self, note):
+        result = note in self.degrees
+        return result
+    
+    def getSuccessors(self):
```

Comment:
I know this is a draft, but certainly there would need to be some more logic here to determine which chords could actually be successors. No reason to place that logic somewhere outside the class instance. Remember, the class instance is responsible for knowing what a chord can `do`, and this would certainly fall into that category.